src/main/java/pl/javastart/task/Circle.java

```diff
@@ -0,0 +1,13 @@
+package pl.javastart.task;
+
+public class Circle {
+    double r;
+
+    public Circle(double r) {
+        this.r = r;
+
+    }
+    double calculateCircleArea() {
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu

W tym przypadku info to: `'METHOD_DEF' should be separated from previous line.` - czyli powinien być dodatkowy enter przed między końcem poprzedniej metody i definicją nowej